api/room/schema_query.py

```diff
@@ -331,10 +332,10 @@ def resolve_analytics_ratios_per_room(
 
     @Auth.user_roles('Admin', 'Default User')
     def resolve_bookings_analytics_count(
-            self, info, start_date, end_date):
+            self, info, start_date, end_date, room_id=None):
```

Comment:
Don't you think it's better to use `kwargs` and leave only `self`, and `info` as the positional arguments?

Reply:
thanx @verenceLola this has been worked on.